sklearn/discriminant_analysis.py

```diff
@@ -247,16 +362,27 @@ class LinearDiscriminantAnalysis(BaseEstimator, LinearClassifierMixin,
     [1]
     """
 
-    def __init__(self, solver='svd', shrinkage=None, priors=None,
-                 n_components=None, store_covariance=False, tol=1e-4):
+    def __init__(
+            self,
+            solver='svd',
+            shrinkage=None,
+            covariance_estimator=None,
```

Comment:
new parameters need to be added to the end of the list, and also, please try to avoid irrelevant changes, such as the new formatting you have here.